pptx_blueprint/__init__.py

```diff
@@ -71,7 +73,7 @@ def _find_shapes(self,
         matched_shapes = []
 
         def _find_shapes_in_slide(slide):
-            return filter(lambda shape: shape.text == f'{{{tag_name}}}', slide.shapes)
+            return filter(lambda shape: shape.name == f'{{{tag_name}}}', slide.shapes)
```

Comment:
Sorry, I don't understand this change. I thought we wanted users to be able to place tags on the slide. shape.name is an object property.